src/draw.js

```diff
@@ -24,6 +24,10 @@ function start(data, e) {
   processDraw(data);
 }
 
+function pointerSelected(ctrl) {
+  return (ctrl && (!ctrl.sparePieceSelected || ctrl.sparePieceSelected === 'pointer'));
```

Comment:
I think this code now exists in 3 places. Time to make a ctrl function it seems.